lib/Config.js

```diff
@@ -385,6 +385,10 @@ function locationConstraintAssert(locationConstraints) {
             assert(locationConstraints[l].isCold === true);
         }
 
+        if (locationConstraints[l].type === 'crr') {
```

Comment:
why do we want to check this?
The code should use `isCrr` to check if this is a car-only location, so we should not check for the type? (type could actually be scality, in that case)